about.html

```diff
@@ -0,0 +1,132 @@
+          </ul>
+        </div>
+
+        </div>
+        <div class="humberger" onclick="openNav()">
+          <i class='fa fa-bars menu'></i>
+        </div>
+
+        <div id="myNav" class="m-menu">
+          <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+          <div class="m-menu-content">
+            <a href="./index.html" onclick="closeNav()">Home</a>
+            <a hre="#" onclick="closeNav()">Speakers</a>
+            <a href="#" onclick="closeNav()">Partners</a>
+            <a href="#" onclick="closeNav()">Join the Program</a>
+            <a href="#" onclick="closeNav()">Main Program</a>
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <div class="agenda-about">
+      <div class="headers">
+        <h2 class="hello-about special-color">"Who are We"</h2>
+        <h1 class="main-topic-about special-color">Group Engineering ENTREPRENEURSHIP & TECHNOLOGY CONFERENCE 2023
+        </h1>
+      </div>
+      <p class="introductory-about">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Et, illo! Rem cupiditate
+        nemo minima voluptate laboriosam deserunt, dignissimos quia incidunt id pariatur odit laborum sunt
+        reprehenderit labore blanditiis temporibus modi.
+      </p>
+      <div class="info-about">
+        <p class="contact-email-about">
+          Please Contact Us with Email for any further questions about
+          this activity!
+          <br /> <b>ytu2021@gmail.com</b>
+        </p>
+      </div>
+    </div>
+  </section>
+  <section class="us">
+    <div class="about-us">
+      <div class="h2-hrline">
+        <h2> Creative Commons Global Summit 2023 Logo</h2>
+        <hr class="hrline">
+      </div>
+
+      <p class="para-us">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Id mollitia
+        magni enim ducimus repudiandae assumenda
+      </p>
+      <img class="img1" src="assets/CC_Global_Summit_2015.png" alt="CC_Global_Summit_2023 logo">
+    </div>
+  </section>
+  <div class="see-past-about">
+    <div class="h-line-past">
+      <h4 class="see-past-h2"> See The Past CC Global Submits</h4>
+      <hr>
+    </div>
+    <p class="see-past-p"> Have a look at the last two years activities</p>
+    <div class="see-21-22">
+      <div class="sub-2021">
+        <h6> 2021 <br> <b> CC Globla Submit 2021</b></h6>
+      </div>
+      <div class="sub-2022">
+        <h6> 2022 <br> <b> CC Globla Submit 2022</b></h6>
+      </div>
+    </div>
+  </div>
+
+  <section class="partners">
+    <div class="h2-hr-part">
+      <h2 class="part-h2">Partners</h2>
+      <hr class="partner-line">
+    </div>
+    <ul class="partners-lst">
+      <li><img src="./assets/fb-logo.png" alt="Facebook"></li>
+      <li><img src="./assets/google-logo.jfif" alt="google"></li>
+      <li><img src="./assets/mozila-logo.png" alt="mozila-logo"></li>
+      <li><img src="./assets/twitter-logo.png" alt="twitter"></li>
+    </ul>
+  </section>
```

Comment:
[OPTIONAL]
- [ ] As it is in the original template, you may consider adding a `orange` overlay on your summit pictures.
Hint: You may consider checking [here](https://stackoverflow.com/questions/18815157/how-to-overlay-image-with-color-in-css#:~:text=Expand%20snippet-,See%20here%20CodePen,-Share) how to add background color on an image

### Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/227735006-7a6ef7a9-56a1-4f58-9ede-f0b2d8dfb903.png)

### Original template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/227735119-c149cc72-33d2-41c2-bd95-c673c419ae8c.png)